script/Ciel/Day 13/13_01_01_CIEL13_3B.txt

```diff
@@ -1424,17 +1530,19 @@ She exists simultaneously as both herself, and as Roa.
 [sha:ddcdb626b320a2d0c8492300dba045dfb8e3d45e]{
 -- Page 227, Offset 33833.
 -- だから、ロアが生きているのにロアであるその子が死んでいては矛盾している」
-As such, because Roa still lives, the very idea that the girl who also bears the name of Roa could die is a contradiction."
+So, if the child who is Roa were to die while Roa himself still lives, it would cause a contradiction."
 }
 [sha:e9680c92fa8750db71a4cba2a61fccd90919ab75]{
 -- Page 228, Offset 33834.
 -- 「この世界はわずかな綻びがあれば、世界が世界自身のためにその綻びを修復します。
-"If our world is torn ever so slightly, the world would mend it for its own sake.
+"If ever a crack forms in the world,
+ the world will patch it over in order to preserve itself.
 }
 [sha:5448f05a0eb259c11bfea87f2a560b654e9641b1]{
 -- Page 228, Offset 33835.
 -- ですから――その子は、ロアという転生する魂が消えないかぎり、永遠にあり続けてしまう。他の誰でもない、この<秩序|せかい>が勝手に『治して』しまうんです」
-Therefore――until the reincarnating soul known as Roa is snuffed out, she will continue to live on in perpetuity, because none other than the <order|w o r l d> itself will restore her on its own accord."
+Therefore――until the reincarnating soul known as Roa is snuffed out, she will continue to live on in perpetuity.
+ If she dies, nothing short of <order|w o r l d   > itself will 'correct' the mistake."
```

Comment:
秩序 hahhh, I'd really like to move it away from "Order" because that word is so overused it makes it seem like there are far more connections between it and オーダー (order) and 人理 (human order) than there is.

Maybe "the balance" would work